### PR TITLE
added get attendees

### PR DESCRIPTION
**Pull request by rrballenger17**

### CODE REVIEW

File: src/com/comeet/Attendees.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    }
+
+    @XmlElement
+    public void setRequiredattendees(List<Person> people) {
+        this.requiredattendees = people;
+    }
+
+    @XmlElement
+    public void setOptionaldattendees(List<Person> people) {
```
Comment (kb000) [inline src/com/comeet/Attendees.java]: I thought we fixed the contract for this typo "optional *d* attendees"

File: src/com/comeet/UsersDao.java [hunk 2]
```diff
@@ -203,5 +204,33 @@ public String appointmentBody(Appointment appt) {
...
+
+        Attendees people = new Attendees();
+        
+        Appointment attAppt = Appointment.bind(service, itemId,
+                        new PropertySet(BasePropertySet.FirstClassProperties));
+       
+        AttendeeCollection req = attAppt.getRequiredAttendees();
+        AttendeeCollection opt = attAppt.getOptionalAttendees();
+        people.setRequiredattendees(attendees(null, attAppt, req));
+        people.setOptionaldattendees(attendees(null, attAppt, opt));
```
Comment (kb000) [inline src/com/comeet/UsersDao.java]: Same typo as above.

File: src/com/comeet/UserService.java [hunk 3]
```diff
@@ -120,6 +120,47 @@
...
+     * </p>
+     * 
+     * @param orgDomain Start of query range.
+     * @param id - meeting ID
+     * @return Meeting info within the query range.
+     * @throws ServiceResponseException If the result is not 200 OK
+     * @throws Exception On an unexpected error.
+     */
+    @POST
+    @Path("/{orgDomain}/meetings/attendees")
```
Comment (kb000) [inline src/com/comeet/UserService.java:135]: Who requested this route? It's not in the apiary docs.

Comment (kb000) [reply src/com/comeet/UserService.java:135]: Oh I see your comment in slack.

File: src/com/comeet/UserService.java [hunk 4]
```diff
@@ -120,6 +120,47 @@
...
+    @Produces("application/json")
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    public Attendees getAttendees(@Context HttpHeaders headers,
+                    @PathParam("orgDomain") String orgDomain, 
+                    @DefaultValue("") @FormParam("id") String id)
```
Comment (kb000) [inline src/com/comeet/UserService.java:140]: Meeting needs to expose the id.
Please fork the api builder spec with these changes.
https://github.com/bookaroom/dev-docs/blob/master/apiary.apib

File: src/com/comeet/UserService.java [hunk 5]
```diff
@@ -120,6 +120,47 @@
...
+            serviceFactory.setAuthContext(authFactory.buildContext(headers));
+
+            // Each call should define new instance of Service and DAO object
+            try (ExchangeService service = serviceFactory.create()) {
+                UsersDao ud = new UsersDao(service);
+                return ud.getAttendees(id);
+            }
+        } catch (AuthContextException e) {
+            // TODO Use OAuth2 for real.
+            e.printStackTrace();
```
Comment (Dairai) [inline src/com/comeet/UserService.java:153]: You should write the exception to the log using the ApiLogger

**Review by kb000 — CHANGES_REQUESTED** [review]

File: src/com/comeet/UserService.java (hunk 4 above)
Comment (kb000) [reply src/com/comeet/UserService.java:140]: er... branch... not fork.

File: src/com/comeet/UserService.java (hunk 5 above)
Comment (Dairai) [reply src/com/comeet/UserService.java:153]: I meant instead of printStackTrace not in addition to. you can change it in another pull request

**Review by kb000 — APPROVED** [review]

**Pull request merged**